lib/zenoss/jsonapi/device_router.rb

```diff
@@ -72,6 +72,24 @@ def set_info(opts = {})
         end
       end
 
+      # @param [String] uid required; device id in Zenoss
+      # @param [String] plugins optional; takes a regular expression
+      # default is an empty string
```

Comment:
what do you think about dropping text specifying defaults here? It's bit redundant, because YARD is smart enough to parse the actual defaults and whether they are required or not directly from the method body

Documentation render would change from **this**:
![image](https://user-images.githubusercontent.com/183976/81664712-a58fd380-9437-11ea-86df-864d0ea71473.png)


to this:

![image](https://user-images.githubusercontent.com/183976/81665943-45019600-9439-11ea-86b4-16bd4da7d1a5.png)